main.py

```diff
@@ -196,14 +197,43 @@ def get_cp_name(or_id: str, ctx: Context) -> str:
     return cp_name
 
 
+def query_params_item_ingested(event: dict, cp_name: str) -> List[Tuple[str, str]]:
+    """ Construct the query parameters to check if an item is already in the MAM.
+
+    A check on S3 object key is always needed.
+
+    A check on md5 is only executed if there is an md5 and
+    the CP is not VRT unless the item is a collateral.
+
+    Returns:
+        List[Tuple[str, str]] -- The query params.
+    """
+    # Check based on the S3 object key
+    query_params = [("s3_object_key", get_from_event(event, "object_key"))]
+
+    # Check based on md5 if the md5 is available and the CP is not VRT
```

Comment:
Comment now slightly out of sync with code.

Reply:
This is a comment on the first commit? It has been changed in the second commit to:

> A check on md5 is only executed if there is an md5 and
> the CP is not VRT unless the item is a collateral.